README.md

```diff
@@ -237,6 +238,23 @@ security:
 #    useAsUsername: "email" # Default is 'email'; custom fields can be used as the username
 #    scopes: "openid, profile, email" # Specify the scopes for which the application will request permissions
 #    provider: "google" # Set this to your OAuth provider's name, e.g., 'google' or 'keycloak'
+#    client:
```

Comment:
maybe we should remove settings.yml from readme and just link to template file at this point.. getting bigger...

Reply:
Okay, I'll change it